_episodes/02-benchmarking.md

```diff
@@ -65,7 +65,8 @@ The `%%timeit` line magic does exactly this in a concise an comfortable manner!
 `%%timeit` first measures how long it takes to run a command one time, then
 repeats it enough times to get an average run-time. Also, `%%timeit` can measure run times without
 the time it takes to setup a problem, measuring only the performance of the code in the cell.
-This way we can trust the outcome better.
+This way we can trust the outcome better. Instead of timing an entire cell,
+it is also possible to time a single line with `%time` or `%timeit`.
```

Comment:
I don't want to raise any gnarly questions about line magics. Most instructors will know this and can make a point out of it if they desire, but I think it distracts from what we want to show.

Reply:
Good point, I'll remove my addition